### PR TITLE
Add some copy changes

### LINKED ISSUE #210: Remove text of 'AWS S3' and replace with a more generic 'S3' in add Replication Repository form
See below 'AWS S3 Bucket Name' and 'AWS S3 Bucket Region'

Screen Shot 2019-05-31 at 8 18 17 PM

### PR DESCRIPTION
**Pull request by eriknelson**
Closes #210

* Keep add cluster modal title consistent with add storage modal. Both
should be "Add X".
* Remove "AWS" from the labels on the add storage form, since they will
not always actually be AWS

### CODE REVIEW

File: src/app/cluster/components/AddClusterModal/index.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ class AddClusterModal extends React.Component<any, any> {
     const { checkConnection, connectionState, name, url, token, mode } = this.props;
 
     return (
-      <Modal isSmall isOpen={this.props.isOpen} onClose={this.handleClose} title="Cluster">
+      <Modal isSmall isOpen={this.props.isOpen} onClose={this.handleClose} title="Add Cluster">
```
Comment (ibolton336) [inline src/app/cluster/components/AddClusterModal/index.tsx]: I think John removes the add because the same modal is used for editing @jwmatthews ?

Comment (eriknelson) [reply src/app/cluster/components/AddClusterModal/index.tsx]: Ah that makes sense. Do we need to remove "Add" from "Add Storage" then? I thought it was odd that they were inconsistent.

**Comment by ibolton336** [comment]
Yeah that’s a good call. We should update both. Is it worth having dynamic text based on the action selected?

**Comment by eriknelson** [comment]
I don't think it's worth dynamic tech. The components are specific to their type anyway.

**Comment by eriknelson** [comment]
The decision was to make these agnostic until we can add in dynamic titles that reflect the action of the modal. Filed an issue to cover the dynamic titles: #257

**Pull request merged**